src/transformers/utils/quantization_config.py

```diff
@@ -379,7 +379,11 @@ class GPTQConfig(QuantizationConfigMixin):
             The exllama config. You can specify the version of the exllama kernel through the `version` key. Defaults
             to `{"version": 1}` if unset.
         cache_block_outputs (`bool`, *optional*, defaults to `True`):
-                Whether to cache block outputs to reuse as inputs for the succeeding block.
+            Whether to cache block outputs to reuse as inputs for the succeeding block.
+        modules_in_block_to_quantize (`List[List[str]]`, *optional*):
+            List list of module names to quantize in the block specified. This argument is useful to exclude certain linear modules from being quantized.
+            The block to quantize can be specified by setting `block_name_to_quantize`. We will quantize each list sequentially. If not set, we will quantize all linear layers.
+            Example: `inside_layer_modules=[["self_attention.query_key_value"], ["mlp.dense_h_to_4h"]]`
```

Comment:
What does each list in the list represent here? E.g. is the first element in the list - `modules_in_block_to_quantize[0]` - the list of layers to quantize for the first block? How does that match with `block_name_to_quantize` which seems to only take a single string i.e. I would expect a single block.

Reply:
Yes, I will detail a little bit more why we are using a list of list. See comment above.